### DISCUSSION TITLE
Allow a user create a league for his friends

### DISCUSSION THREAD

**Discussion by kirhgoff** [monkeytypegame/monkeytype#1508] [Ideas] [answered]
I started learning 10 finger typing and I grew from 10 wpm to ~40 wpm. I checked leaderboards and found that there are people with 200 wpm and there is no way I can compete. But it could be good fun to challenge my colleagues or friends and compete with them. 

I thought it would be nice to be able to create a league, invite some friends by email so you can have an independent leaderboard with only them and observe their best results.

**Comment by Miodec** [comment, marked as answer]
Could be interesting, ill think about it. There is also a multiplayer mode being worked on at the moment that also will allow you to compete with your friends
